DigitalExcellence/dex-frontend
# Added user back to project-overview page.

## Description

Added user back to the project-overview page

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to project-overview page
2. See that it now shows users on the right column

## Link to issue

Closes: #207
